capstone/src/main/java/com/google/sps/data/RecommenderAlgorithm.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps.data;
+ 
+import com.google.sps.data.Room;
+import com.google.sps.data.User;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Entity;
+import java.util.HashMap;
+import java.util.ArrayList;
+import java.util.List;
+import java.lang.Iterable;
+ 
+public class RecommenderAlgorithm {
+ 
+    public static ArrayList<Room> recommendRooms(User user) {
```

Comment:
Recommend adding a comment that explains (briefly) how this works, or link to your design doc.